public/components/GeoPlot.vue

```diff
@@ -838,16 +773,18 @@ export default Vue.extend({
     getInterestBounds(area: Area): LatLngBoundsLiteral {
       const xDistance = (this.drillDownState.numCols - 1) / 2;
       const yDistance = (this.drillDownState.numRows - 1) / 2;
-      const tileWidth = area.coordinates[1][1] - area.coordinates[0][1];
-      const tileHeight = area.coordinates[1][0] - area.coordinates[0][0];
+      const tileWidth =
+        area.info.coordinates[1][1] - area.info.coordinates[0][1];
```

Comment:
I see that you need the tile width and height here and in the _drilldown_ component. 
Maybe you could add it to the class `CoordinateInfo`, so the next reader doesn't need to check if `coordinates[0]` is _Lat_ or _Long_.
You could also add `lat` and `long` as properties for a more straightforward use; thus, only `util/rendering/coordinates.ts` will manage which one is which.

To resume, something like this:
```js
export abstract class CoordinateInfo {
  coordinates: LatLngBoundsLiteral;
  lat: number,
  long: number,
  height: number,
  width: number,
  ...
}
```